.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "KPP/standalone"]
+	path = KPP/standalone
+	url = https://github.com/geoschem/KPP-Standalone
+	tag = "GC_Interface"
```

Comment:
Will this code be updated in the future? If yes, I wonder if the tag should have a version number in it so it is easy to keep the same name but not have to move the tag.

Reply:
Thanks @lizziel.  The code might be updated in the future, but perhaps not corresponding to each GEOS-Chem version number.  For example, we might add a better command-line parser to the standalone but that would not have anything to do with the chemical mechanism.  I am open to having the version number in the tag.

Reply:
I just realized we don't even need to include tag since the hash is committed. For example, in GCHP we only put path and url: https://github.com/geoschem/GCHP/blob/main/.gitmodules. Same for GC-Classic: https://github.com/geoschem/GCClassic/blob/main/.gitmodules. Why is tag specified?

Reply:
> We should link to a tag here rather than a branch.

@lizziel: I added it based on one of your earlier comments.  I can take out the tag if it's not needed.

Reply:
@lizziel: "tag" has been removed from `.gitmodules` in commit 12966062.

Reply:
Changelog also updated in https://github.com/geoschem/geos-chem/pull/2588/commits/8bd451fb233c9189794439fe0ddd476023a89c5a